docs/src/index.md

```diff
@@ -46,6 +46,8 @@ codes such as [Abinit](https://www.abinit.org/),
       DFTK-based calculator implementation for ASE.
     - Read data in [ETSF Nanoquanta](https://doi.org/10.1016/j.commatsci.2008.02.023) format.
 
+Missing a feature? Look for an open issue or create one [here](https://github.com/JuliaMolSim/DFTK.jl/issues).
```

Comment:
here is generally bad as a name for a link. What if the link is broken ... I would link `or [create a new one](....).`